scripts/sequoia-nixbld-user-migration.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eo pipefail
 
+# stock path to avoid unexpected command versions
+PATH="$(/usr/bin/getconf PATH)"
```

Comment:
This should yield a standard system PATH. For context, here's what it outputs on mine:
```console
$ getconf PATH
/usr/bin:/bin:/usr/sbin:/sbin
```